src/PageContainer/component.js

```diff
@@ -6,14 +6,14 @@ import urlify from "../_utils/urlify"
 // so we need to normalize according to collection data
 const splatToUrl = (string) => ("/" + urlify(string))
 
-const isDevelopment = process.env.NODE_ENV !== "production"
-const isClient = typeof window !== "undefined"
-const isDevelopmentClient = isDevelopment && isClient
+const isDevelopment = () => process.env.NODE_ENV !== "production"
+const isClient = () => typeof window !== "undefined"
+const isDevelopmentClient = () => isDevelopment() && isClient
 
 let catchLinks
 let browserHistory
 
-if (isClient) {
+if (isClient()) {
   catchLinks = require("../_utils/catch-links").default
   browserHistory = require("../client").browserHistory
```

Comment:
@MoOx here is why I need to clean node cache. I need to refresh `browserHistory`